lib/editor/components/map/PatternsLayer.js

```diff
@@ -243,6 +242,7 @@ class PatternSegment extends PureComponent<PatternSegmentProps> {
     return (
       <Polyline
         // React leaflet coordinates are [lat, lon]
+        key={Math.random()}
```

Comment:
I remember there being a reason why we didn't use uuid but regrettably I don't remember. I'll add a comment saying we need to look into this.

It seems like this (anti) pattern is copied from other parts of datatools so I'll add an issue instead of a comment.